internal/exec/vendor_utils.go

```diff
@@ -83,7 +83,7 @@ func ExecuteVendorPullCommand(cmd *cobra.Command, args []string) error {
 
 	// Check `vendor.yaml`
 	vendorConfig, vendorConfigExists, foundVendorConfigFile, err := ReadAndProcessVendorConfigFile(cliConfig, cfg.AtmosVendorConfigFileName)
-	if err != nil {
+	if vendorConfigExists && err != nil {
```

Comment:
I think we should log a warning to make it easier to debug

Reply:
@coderabbitai any suggestions?

Reply:
@coderabbitai please make the suggestion

Reply:
@Cerebrovinny ☝️

Reply:
Thank you @osterman this is resolved now